gen/definitions/classes.yaml

```diff
@@ -252,6 +253,7 @@ tagAnnotation:
     - "uni/tn-{name}/BD-{name}/annotationKey-[{key}]"
     - "uni/tn-{name}/ctx-{name}/annotationKey-[{key}]"
     - "Too many DN formats to display, see model documentation for all possible parents of [tagAnnotation](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagAnnotation/overview)."
+  test_type: "apic"
```

Comment:
do we need this? If we change the example class to fvCtx (from fvAEPg) wouldn't this fix the same issue?

Reply:
We don't need this anymore because we don't run CI on cloud APICs. However, this test was failing when I ran it on a cloud apic.